packages/core/src/components/dropdown-menu/dropdown-menu.tsx

```diff
@@ -94,26 +81,24 @@ export class DropdownMenu {
         }
       }
     }
-  }
 
-  openPopup() {
-    this.host.setAttribute("tabindex", "-1");
-    this.host.addEventListener("keydown", this.keyDownListener);
-    this.host.addEventListener("focusout", this.focusOutListener);
-    this.button.setAttribute("aria-expanded", "true");
-    this.tabbables.forEach((tabbable) =>
-      tabbable.addEventListener("click", this.escape)
-    );
-  }
-
-  closePopup() {
     this.host.removeEventListener("keydown", this.keyDownListener);
     this.host.removeEventListener("focusout", this.focusOutListener);
-    this.button.setAttribute("aria-expanded", "false");
-    this.host.removeAttribute("tabindex");
-    this.tabbables.forEach((tabbable) =>
-      tabbable.removeEventListener("click", this.escape)
-    );
+    this.button.setAttribute("aria-expanded", this.open ? "true" : "false");
+    if (this.open) {
+      this.host.addEventListener("keydown", this.keyDownListener);
```

Comment:
`this.host.addEventListener`, `this.host.setAttribute` en `this.host.removeAttribute` kan met het functionele component `<Host>` worden geregeld:

```ts
render() {
  return (
    <Host onFocusout={this.focusOutListener} tabindex={this.open ? "-1" : undefined}>
      ...
    </Host>
  );
}

Geen halszaak natuurlijk, maar als we dan toch met code refactoring bezig zijn is het wel een uitgelezen kans om hier wat verbeteringen mee te pakken.

De release is morgen pas, wat denk jij?

Reply:
https://github.com/dso-toolkit/dso-toolkit/commit/f7e5416b6797d725d23f7e3e7d23d85b2d1e3a4b

Reply:
💐